baseball.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+        <head>
+            <meta charset="utf-8">
+            <title>number baseball</title>
+        </head>
+
+        <body>
+            <button type="button" id="start">게임시작</button>
+            <input type="number" name="usernnn" id="nnn" min="100" max="999" title="세자리숫자입력">
+            <button type="submit" name="enter" id="enter">Enter</button>
+            
+            <button id="restart">게임재시작</button>
+            <script>
+                var btn = document.getElementById('start');
+                btn.addEventListener('click',rnnn);
+
+                function rnnn() {
+                    console.log(((Math.random()*0.9)+ 0.1).toFixed(3)*1000)
+                };
```

Comment:
```suggestion
                }
```
`function`키워드로 시작하는 함수 선언문과, `if`, `for`과 같은 조건문/반복문 뒤에는 세미콜론을 붙이지 않습니다.
어떤 위치에 세미콜론을 써야할지 조사해보세요.